### DISCUSSION TITLE
Bar Chart Corner Radius on Grouped Bars

### DISCUSSION THREAD

**Discussion by scottwillman** [naver/billboard.js#2231] [Q&A]
Looks like bar corner radius isn't implemented on grouped bars: https://github.com/naver/billboard.js/blob/37b698d85fb7c83d044ef706c931ed0d0b306118/src/ChartInternal/shape/bar.ts#L124

Is this a heavy lift? A quick scan of the code seems like it could be as simple as only applying the rounding function to the last bar in the group. But, I assume if it were that simple it'd be in there already :). Can I request this feature? Would be a nice bit of polish on some cool charts!

**Comment by netil** [comment]
@scottwillman sorry to be late answering this, bar radius for stacked(or grouped) will be considered for future release 😃 
- https://github.com/naver/billboard.js/issues/2428

**Comment by netil** [comment]
this feature was released in v3.3
- https://naver.github.io/billboard.js/demo/#BarChartOptions.BarRadius
